docs/src/PolyhedralGeometry/Polyhedra/pg_polyhedra_polymake.md

```diff
@@ -0,0 +1,55 @@
+```@meta
+CurrentModule = Oscar
+```
+
+```@setup oscar
+using Oscar
+```
+
+```@contents
+Pages = ["pg_polyhedra_polymake.md"]
+```
+
+
+# `Polyhedron` and `polymake`'s `Polytope`
+
+Many polyhedral computations are done through `polymake`.
```

Comment:
The user may not know what "polymake" is at this point. I'd suggest to at least insert one brief sentence explaining it, and perhaps a reference to (to be added) entry in the bibliography for polymake, and/or a link to the polymake homepage?